### DISCUSSION TITLE
Best way to update own repository (template) to latest version

### DISCUSSION THREAD

**Discussion by Catbirby** [upptime/upptime#110] [Q&A] [answered]
I was looking for what would be the best/easiest way to update my version of the template to the latest version that is present in the upptime repository

**Comment by Shulert** [comment]
You have to manually run the Workflows (not sure which one). Or, you can wait for it's scheduled run.

**Comment by AnandChowdhary** [comment, marked as answer]
You can manually run "Update Template CI".
